core/shared/src/main/scala/org/http4s/RangeUnit.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2013 http4s.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.http4s
+
+import org.http4s.util.{Renderable, Writer}
+
+object RangeUnit {
```

Comment:
I'm not sure why this is a pure addition.  What is it in 0.23?

Reply:
https://github.com/http4s/http4s/blob/main/core/src/main/scala/org/http4s/RangeUnit.scala ?

Reply:
Oh, it showed up as a new file because we lost the attribution in the header, and that was as substantial as the file itself.